src/main/java/uk/gov/pay/products/validations/ProductRequestValidator.java

```diff
@@ -27,8 +27,13 @@ public ProductRequestValidator(RequestValidations requestValidations) {
                 FIELD_GATEWAY_ACCOUNT_ID,
                 FIELD_PAY_API_TOKEN,
                 FIELD_NAME,
-                FIELD_PRICE,
-                RETURN_URL);
+                FIELD_PRICE);
+
+        if(!errors.isPresent()){
+            if(!requestValidations.checkIfExists(payload, FIELD_RETURN_URL).isPresent()) {
```

Comment:
😖 seriously ? Do you really need requestValidations to check if the `payload` having a field named `FIELD_RETUEN_URL`?

Reply:
👍

Reply:
Fair enough, I shall change it 👍